src/components/ShoppingCart/index.tsx

```diff
@@ -5,22 +5,22 @@ import { useGetItemQuantityQuery } from "@/redux/api/cartApi";
 import { userData } from "@/store/reducer/userSlice";
 import { skipToken } from "@reduxjs/toolkit/dist/query";
 import EmptyCart from "./EmptyCart";
-import { ICartItem } from "@/interfaces/cartItem";
+import { ICartItem, ICartItem2 } from "@/interfaces/cartItem";
 
 const ShoppingCart = () => {
   const isCartOpen = useStoreSelector((state) => state.buttonToggle.isCartOpen);
   const currentUserId = useStoreSelector(userData).userId;
   const { data } = useGetItemQuantityQuery(currentUserId ? currentUserId : skipToken);
   const quantity = data?.length;
 
-  const cart: ICartItem[] = data?.map((item: any) => {
+  const cart: ICartItem[] = data?.map((item: ICartItem2) => {
```

Comment:
If we use `item:ICartItem`, will it be any different from using `item: ICartItem2`?